code/modules/cargo/packs/emergency.dm

```diff
@@ -62,7 +62,7 @@
 	crate_type = /obj/structure/closet/crate/internals
 
 /datum/supply_pack/emergency/medicalemergency
-	name = "Emergency Medical Supplies" //Almost all of this can be ordered seperatly for a much cheaper price, but the HUD increases it.
+	name = "Emergency Medical Supplies" //Almost all of this can be ordered seperately for a much cheaper price, but the HUD increases it.
```

Comment:
```suggestion
	name = "Emergency Medical Supplies" //Almost all of this can be ordered separately for a much cheaper price, but the HUD increases it.
```